lib/less-parser.js

```diff
@@ -114,9 +114,9 @@ export default class LessParser extends Parser {
       // eslint-disable-next-line
       delete this.current.nodes;
 
-      if (node.selector.indexOf('!important') >= 0) {
+      if (node.selector.match(/!important/i)) {
```

Comment:
should be `.test` as you don't care about the result

Reply:
good catch

Reply:
resolved